src/styles/blocks/catalog.scss

```diff
@@ -18,8 +18,16 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: $animationDuration transform ease, $animationDuration color ease;
+
```

Comment:
I'm not sure abot the order of it.. I think first goes color and than time. but if it work its okay=)